src/FSharp.Editing.VisualStudio/Common/Utils.fs

```diff
@@ -111,12 +114,9 @@ type SnapshotSpan with
     member inline x.EndLineNum  = x.Snapshot.GetLineNumberFromPosition x.End.Position
     member inline x.EndColumn = x.End.Position - x.EndLine.Start.Position
 
-    member x.ModStart (num) = SnapshotSpan(SnapshotPoint (x.Snapshot, x.Start.Position + num), x.End)
-    member x.ModEnd (num) = SnapshotSpan(x.Start, (SnapshotPoint (x.Snapshot,x.End.Position + num)))
-
-    member x.ModBoth m1 m2 =
-        SnapshotSpan(SnapshotPoint (x.Snapshot, x.Start.Position + m1),
-                     SnapshotPoint (x.Snapshot, x.End.Position + m2))
+    member x.ModStart num = VS.Snapshot.mkSpan (x.Start + num) x.End 
```

Comment:
I just think that
```fsharp
VS.Snapshot.mkSpan (x.Start + m1, x.End + m2)
```
is better than
```fsharp
VS.Snapshot.mkSpan (x.Start + m1) (x.End + m2)
```
because there are less parens.